onnx/proto_utils.h

```diff
@@ -3,6 +3,8 @@
 #include <google/protobuf/io/coded_stream.h>
 #include <google/protobuf/io/zero_copy_stream_impl_lite.h>
 
+#include "onnx/onnx_pb.h"
```

Comment:
Why add this?

Reply:
Because it is needed... It used some function under onnx_pb.h, therefore if you include proto_utils.h without include onnx_pb.h, there will be a compile error.

Reply:
I don't think so... I guess because in some other files, we didn't correctly import #onnx/onnx_pb.h"

Reply:
@houseroad "class AttributeProto" is defined under onnx/onnx_pb.h, and used here...